frontend/src/store/uam/reducer.ts

```diff
@@ -2,15 +2,17 @@ import { createReducer } from '@reduxjs/toolkit';
 import { DataStatus } from 'common/enums/enums';
 import { UsersGetAllItemResponseDto } from 'common/types/types';
 
-import { getUsers } from './actions';
+import { deleteUser, getUsers } from './actions';
 
 type State = {
   dataStatus: DataStatus;
+  isDeleted: DataStatus;
```

Comment:
Why do we need `isDeleted`? We can change values of `dataStatus`

Reply:
`userDeleteDataStatus`